package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "lint": "eslint --ignore-path .gitignore ./",
     "lint:fix": "eslint --fix --ignore-path .gitignore ./",
-    "test": "jest --coverage",
+    "test": "npm run lint && jest --coverage",
```

Comment:
Its not a big deal, but we use `pretest` in our other projects to run the linting step. Don't feel too strongly about it one way or the other.